doc/contribute/contributor_expectations.rst

```diff
@@ -87,27 +87,39 @@ Changes which require an RFC proposal include:
 Maintainers have the discretion to request that contributors create an RFC for
 PRs that are too large or complicated.
 
+.. _pr_requirements:
+
 PR Requirements
 ***************
 
 - Each commit in the PR must provide a commit message following the
   :ref:`commit-guidelines`.
 
+- No squash, fixup or merge commits are allowed, explicit or not.
+
 - The PR description must include a summary of the changes and their rationales.
 
 - All files in the PR must comply with :ref:`Licensing
   Requirements<licensing_requirements>`.
 
-- Follow the Zephyr :ref:`coding_style` and :ref:`coding_guidelines`.
+- The code must follow the Zephyr :ref:`coding_style` and :ref:`coding_guidelines`.
+
+- If images are included in the PR, they must be reasonably sized and compressed
+  to be stored in an efficient manner.
```

Comment:
Good idea, can we add a reference of some compression thing? @kartben @gmarull I think there's a few that you usually suggest? tinypng, shrinkmypng, pimpmypng or something, can't remember